### PR TITLE
automated: linux: add systemd-analyze

### PR DESCRIPTION
**Pull request by roxell**
Add systemd-analyze to analyze the different boot times, firmware, bootloader, kernel and userspace.

### CODE REVIEW

**Comment by chase-qi** [comment]
LGTM.

**Review by ivoire — APPROVED** [review]

File: automated/linux/systemd-analyze/systemd-analyze.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+done
+
+create_out_dir "${OUTPUT}"
+
+pkgs="systemd"
+install_deps "${pkgs}" "${SKIP_INSTALL}"
+
+command -v systemd-analyze
+exit_on_fail "systemd-analyze not in ${PATH} or not installed"
+systemd-analyze | tee "${OUTPUT_FILE}"
```
Comment (bhcopeland) [inline automated/linux/systemd-analyze/systemd-analyze.sh:37]: systemd-analyze time is used by default, but you should define <time> just incase it ever changes.

File: automated/linux/systemd-analyze/systemd-analyze.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2024 Linaro Ltd.
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: systemd-analyze
+    description: |
+        The systemd-analyze analyzes the boot times. It's boot times
```
Comment (bhcopeland) [inline automated/linux/systemd-analyze/systemd-analyze.yaml:7]: *measures*

Comment (roxell) [reply automated/linux/systemd-analyze/systemd-analyze.yaml:7]: Thank you @bhcopeland this PR https://github.com/Linaro/test-definitions/pull/508 should fix it

**Pull request merged**